runtime/interpreter/interpreter.go

```diff
@@ -2909,20 +2914,23 @@ func (interpreter *Interpreter) capabilityCheckFunction(
 				panic(errors.NewUnreachableError())
 			}
 
-			targetStorageKey, authorized :=
-				interpreter.getCapabilityFinalTargetStorageKey(
-					addressValue,
+			address := addressValue.ToAddress()
+
+			targetStorageKey, authorized, err :=
+				interpreter.GetCapabilityFinalTargetStorageKey(
+					address,
```

Comment:
is this expected to be called with addressValue.ToAddress() instead of addressValue ?

Reply:
The signature of `GetCapabilityFinalTargetStorageKey` changed, it only needs a `common.Address` now